modules/ding_provider/ding_provider.admin.inc

```diff
@@ -27,5 +27,19 @@ function ding_provider_admin_overview() {
     }
   }
 
-  return $build;
+  $build['wipe_provider_users_threshold'] = array(
```

Comment:
I would like this variable to be prefixed with module name (ding_provider_users_wipe_threshold). It would make it easier to debug later on in the DB.